Classes/Ttree/ContentRepositoryImporter/DataType/Date.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Ttree\ContentRepositoryImporter\DataType;
+
+/*                                                                                  *
+ * This script belongs to the TYPO3 Flow package "Ttree.ContentRepositoryImporter". *
+ *                                                                                  */
+
+use TYPO3\Flow\Annotations as Flow;
+
+/**
+ * Date Data Type
+ */
+class Date extends DataType
+{
+
+    protected function initializeValue($value)
+    {
+        if ($value instanceof \DateTimeInterface) {
+            // $value is already a DateTime object, use it directly
+            $this->value = clone $value;
+        } elseif (is_numeric($value)) {
+            // $value is UNIX timestamp
+            if ($timestamp = (int)$value) {
+                $this->value = new \DateTime();
+                $this->value->setTimestamp($timestamp);
+            } else {
+                $this->value = null;
+            }
+        } elseif (is_string($value)) {
+            // fallback, try to parse $value as a date string
+            $this->value = new \DateTime($value);
```

Comment:
@remuslazar it's your code... could you please give Dominique some feedback on that?